grants/milestone_deliveries/season_2/package_manager/milestone1-part2-details.md

```diff
@@ -0,0 +1,113 @@
+# Radicle Package Manager - Part 2 - Research Notes
+
+This summary report extends [the first part of research](./milestone1-part1-details.md). This second part of the research focuses on [Part 4 of Milestone 1 of the research grant](https://radicle.community/t/grant-application-package-manager/3122#milestone-1-npm-compatibility-12): Verdaccio integration. As the research unfolded, it became clear that it would be best to focus on part 4 (Verdaccio integration) first, because the lessons learned from this research informs the discussion and research for part 2 (package signing) and part 3 (package uploads).
```

Comment:
We should change "Part 4" to "Part 3" I guess?

See note here on us dropping one of the parts of milestone 1:
https://github.com/radicle-dev/radicle-grants/pull/90/files#r1064157879

Reply:
I'm not sure how to make this less confusing. The big takeaway here is that the work did no unfold in the same order as the original grant estimate.

Grant Estimate:
1. Package code
2. Package signing
3. Package Upload
4. Verdaccio Integration

Actual research:
1. Package code
2. Verdaccio Integration
3. Package signing

The 'package upload' step was going to focus on Filecoin or Areweave uploads, but the results from the 'package signing' research made it clear that it is not a great idea at this time without a package signing scheme.

Reply:
Similar point to below:
https://github.com/radicle-dev/radicle-grants/pull/90#discussion_r1064810133

It sounds like we might want to add a package signing scheme to Radicle

Would you agree?

Reply:
It's really nice that Radicle can be used as an alternative to npm and GitHub in its current implementation.

But being able to distribute packages over any medium (npm, GitHub packages, Filecoin, Arweave, etc) would be a really nice (and industry standard) feature. Radicle needs to have a package signing scheme in order to do that.

So yes, I would agree that Radicle should consider adding a package signing scheme in the roadmap for future development.